static/js/components/widgets/ResourcePickerDialog.tsx

```diff
@@ -189,30 +175,6 @@ export default function ResourcePickerDialog(props: Props): JSX.Element {
                 </NavLink>
               </NavItem>
             ))}
-            {dropdownTabs.length > 0 && (
-              <Dropdown
```

Comment:
😭

Reply:
well, it will always be there in `git blame` for us 😄